src/test/kotlin/ContainerUnitTest.kt

```diff
@@ -237,6 +238,60 @@ class ContainerUnitTest {
         }
     }
 
+    private val getContentFromRCCases = listOf(
+        "sng_book_various_content_rc",
+        "sng_book_various_content_rc.zip"
+    )
+
+    @Test
+    @Throws(Exception::class)
+    fun getContentFromRC() {
+        getContentFromRCCases.forEach {
+            val classLoader = this.javaClass.classLoader
+            val resource = classLoader.getResource(it)
+            val rcPath = File(resource!!.toURI().path)
+            val tempDir = createTempDir()
+            val rc = ResourceContainer.load(rcPath)
+            val wavContent = rc.getProjectContent(
+                    projectIdentifier = "sng", extension = "wav"
+            )
+
+            assertNotNull(wavContent)
+            assertEquals(2, wavContent!!.streams.size)
+
+
+            wavContent.streams.forEach { entry ->
+                tempDir.resolve(File(entry.key).name).outputStream().use { output ->
+                   output.write(entry.value.read())
+                }
+            }
+
+            val usfmContent = rc.getProjectContent(extension = "usfm")
+
+            assertNotNull(usfmContent)
+            assertEquals(1, usfmContent!!.streams.size)
+
+            usfmContent.streams.forEach { entry ->
+                tempDir.resolve(File(entry.key).name).outputStream().use { output ->
+                    output.write(entry.value.read())
+                }
+            }
+            rc.close()
```

Comment:
what are we testing here?

Reply:
I moved that to a new method. It tests all streams are closed properly.